src/main/java/com/currencycloud/client/model/TermsAndConditionsAcceptance.java

```diff
@@ -0,0 +1,23 @@
+package com.currencycloud.client.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+import java.util.Date;
+import java.util.UUID;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class TermsAndConditionsAcceptance {
+  private UUID acceptanceId;
```

Comment:
For consistency with other entities the Id should be a String rather than a UUID